data/test/scenarios/wml_tests/ScenarioWML/EventWML/ActionWML/DirectActionsWML/harm_unit_kill_split_experience_no_attack_defend_kill.cfg

```diff
@@ -0,0 +1,60 @@
+# wmllint: no translatables
+
+#####
+# API(s) being tested: [harm_unit]
+##
+# Actions:
+# Set Alice, Charlie and Dave to a known number of hitpoints.
+# Set Bob to a known but higher number of hitpoints.
+# Use [harm_unit] with Alice as harmer, the others as victims, and damage equal to Charlie's hitpoints.
+##
+# Expected end state:
+# Alice gets 17 experience points, Bob gets 1, Charlie and Dave die.
```

Comment:
A comment here about "no" having no effect in the comma-separated list would help future readers.